tests/roles/pull_openstack_configuration/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+dpa_dir: "~/data-plane-adoption"
+os_diff_data_dir: "~/os-diff"
```

Comment:
These will pollute user's home directory... Is there a reason not to do what i suggested earlier?

```
# change os_diff_data_dir into:
os_diff_dir: tmp/os-diff

# change dpa_dir into:
os_diff_data_dir: tmp/os-diff/data
```

This also includes descriptive naming of the variable and not creating directories with test data that aren't yet in the `.gitignore` file. The `playbooks/tmp` directory is already in gitignore. So i think the above should "just work" both in context of user environments and CI.

Reply:
Note these paths are not in `/tmp`, they are relative to the playbook, so in effect they are `<data_plane_adoption_repo>/tests/playbooks/tmp`.